src/System.Windows.Forms/src/System/Windows/Forms/ContainerControl.cs

```diff
@@ -940,26 +941,43 @@ protected override void OnParentChanged(EventArgs e)
         ///  PerformAutoScale are passed as follows:
         ///  1. If AutoScaleDimensions are set, includedBounds is set to true.
         ///  2. If a font change occurred, excludedBounds is set to true.
+        ///
+        /// 'scalingByFontChanged' parameter help distinguish the scaling by 'ResumeLayout'
+        /// or 'OnFontChanged' event. Scaling by 'OnFontChanged' event does not need to scale
+        /// child container controls as they receive their own 'OnFontChanged' message.
```

Comment:
Use [correct xml-doc](https://docs.microsoft.com/dotnet/csharp/programming-guide/xmldoc/param), `<param name="scalingByFontChanged">...</param>`
```suggestion
        /// <param name="scalingByFontChanged">Indicates whether the scaling was caused by 'ResumeLayout'
        /// or 'OnFontChanged' event. Scaling by 'OnFontChanged' event does not need to scale
        /// child container controls as they receive their own 'OnFontChanged' message.</param>
```